src/cow-react/modules/wallet/web3-react/connectors/Trezor/index.tsx

```diff
@@ -0,0 +1,180 @@
+import { Actions, Connector } from '@web3-react/types'
+import Web3ProviderEngine from 'web3-provider-engine'
+import TrezorConnect from '@trezor/connect-web'
+import transformTypedData from '@trezor/connect-plugin-ethereum'
+
+global.process = { browser: true } as any
+const TrezorSubprovider = require('@0x/subproviders/lib/src/subproviders/trezor').TrezorSubprovider
+const RPCSubprovider = require('@0x/subproviders/lib/src/subproviders/rpc_subprovider').RPCSubprovider
+
+interface LedgerConstructorArgs {
+  actions: Actions
+  onError?: () => void
+  options?: TrezorOptions
+}
+
+interface TrezorOptions {
+  chainId?: number
+  url?: string
+  manifestEmail: string
+  manifestAppUrl: string
+}
+
+function parseChainId(chainId: number | string) {
+  return Number.parseInt(String(chainId), 16)
+}
+
+const DEFAULT_MANIFEST_EMAIL = ''
```

Comment:
Should we specify a value here?

Reply:
I see, we pass it from `initializeConnector`, then I think we don't need these default parameters at all